common/src/main/scala/io/phdata/pipeforge/common/jdbc/DataType.scala

```diff
@@ -8,7 +8,8 @@ object DataType extends Enumeration {
     column match {
       case Column(_, _, JDBCType.NUMERIC, _, _, p, s) if s > 0               => JDBCType.DECIMAL
       case Column(_, _, JDBCType.NUMERIC, _, _, p, s) if s == 0 && p > 19    => JDBCType.DECIMAL
-      case Column(_, _, JDBCType.NUMERIC, _, _, p, s) if s == 0 && p > 10    => JDBCType.BIGINT
+      case Column(_, _, JDBCType.NUMERIC, _, _, p, s) if s == 0 && p >= 10   => JDBCType.BIGINT
+      case Column(_, _, JDBCType.NUMERIC, _, _, p, s) if s == 0 && p > 10    => JDBCType.INTEGER
```

Comment:
I don't see when this line would trigger?

Reply:
We ran into an issue at a client where when an Oracle table is created with NUMBER(10,0) and the value was larger than max int sqoop would fail to map the recordset to INTEGER and throw a number overflow exception.  This aims to look at the source definition and if it is 9 digits then map to Integer over 10 or larger will be mapped to bigint.

Reply:
Ah but I see, I didnt flip the check.  I will fix.